colibre/auto_plotter/grain_size_comparison.yml

```diff
@@ -0,0 +1,175 @@
+stellar_mass_v_size_ratio:
+  type: "scatter"
+  legend_loc: "lower right"
+  x:
+    quantity: "apertures.mass_star_50_kpc"
+    units: Solar_Mass
+    start: 3e7
+    end: 3e12
+    log: true
+  y:
+    quantity: "derived_quantities.dust_small_to_large_ratio_50_kpc"
+    units: dimensionless
+    start: 0.1
+    end: 10
+    log: true
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 3e7
+      units: Solar_Mass
+    end:
+      value: 3e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar mass vs Small-to-Large Ratio
+    caption: The dust mass ratio of small to large grains in all gas averaged of 50 kpc in galaxies. Compared to values derived from observation by Relano et al 2020.
```

Comment:
I'm not sure I understand the "averaged of 50 kpc" bit of the caption. Is this the average over a 50 kpc aperture?

Reply:
its just the mass ratio aggregated over the aperture, so 'average' isn't really accurate here, changed to reflect